examples/commit-rollback-handlers/commit_rollback_handlers.description

```diff
@@ -1,4 +1,6 @@
-// Often code needs to get executed depending on whether a transaction committed.
-// Testing the result of the commit within the transaction statement works, but inconvenient from a modularity perspective, particularly when you want to undo changes on rollback.
+// Often the code needs to get executed depending on whether a transaction is committed.
```

Comment:
We don't need "the" before code right? Given that there can be multiple code blocks.